[2025-01-22T09:02:00.050+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-22T09:02:00.059+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9868205040_2025-01-22T09-01-38-503093Z.make_api_call scheduled__2025-01-22T09:01:38.503093+00:00 [queued]>
[2025-01-22T09:02:00.062+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9868205040_2025-01-22T09-01-38-503093Z.make_api_call scheduled__2025-01-22T09:01:38.503093+00:00 [queued]>
[2025-01-22T09:02:00.062+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-22T09:02:00.068+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): make_api_call> on 2025-01-22 09:01:38.503093+00:00
[2025-01-22T09:02:00.072+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=22391) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-22T09:02:00.073+0000] {standard_task_runner.py:72} INFO - Started process 22392 to run task
[2025-01-22T09:02:00.074+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9868205040_2025-01-22T09-01-38-503093Z', 'make_api_call', 'scheduled__2025-01-22T09:01:38.503093+00:00', '--job-id', '1162', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmpmtepsec0']
[2025-01-22T09:02:00.074+0000] {standard_task_runner.py:105} INFO - Job 1162: Subtask make_api_call
[2025-01-22T09:02:00.097+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9868205040_2025-01-22T09-01-38-503093Z.make_api_call scheduled__2025-01-22T09:01:38.503093+00:00 [running]> on host 680cb867086b
[2025-01-22T09:02:00.135+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9868205040_2025-01-22T09-01-38-503093Z' AIRFLOW_CTX_TASK_ID='make_api_call' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T09:01:38.503093+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-22T09:01:38.503093+00:00'
[2025-01-22T09:02:00.135+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-22T09:02:00.143+0000] {logging_mixin.py:190} INFO - API call to http://192.168.79.109:8001/call completed with status code 200
channelId: demo-scheduled-from-ui-worldlink26c12ed6-13ce-46fb-83dc-cfe8a2cce6fb
[2025-01-22T09:02:00.143+0000] {python.py:240} INFO - Done. Returned value was: 200
[2025-01-22T09:02:00.155+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-22T09:02:00.155+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9868205040_2025-01-22T09-01-38-503093Z, task_id=make_api_call, run_id=scheduled__2025-01-22T09:01:38.503093+00:00, execution_date=20250122T090138, start_date=20250122T090200, end_date=20250122T090200
[2025-01-22T09:02:00.166+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-22T09:02:00.179+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-22T09:02:00.180+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
